packages/video_player/video_player/pubspec.yaml

```diff
@@ -3,7 +3,7 @@ description: Flutter plugin for displaying inline video with other Flutter
   widgets on Android and iOS.
 author: Flutter Team <flutter-dev@googlegroups.com>
 version: 0.10.2+6
```

Comment:
Version bump and changelog are in the second PR. I separated them because the usage of the platform interface will depend on this and the tests will fail. Is this necessary? And does the first part also need updates to the changelog and version?

Reply:
Can I put it all in one PR?

Reply:
You won't be able to depend on the platform interface in this PR since you won't be able to upload it to pub until this PR lands